cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -118,6 +118,33 @@ def _generate_template(cluster, capsys):
     return generated_template
 
 
+@pytest.mark.parametrize(
+    "config_file_name",
+    [
+        "centos7.slurm.full.yaml",
```

Comment:
Instead of create new config here, can we utilize the config in example_config: see example in `test_cluster_lambda_functions_vpc_config` 
input_yaml = load_yaml_dict(f"{EXAMPLE_CONFIGS_DIR}/{config_file_name}")
We can see if we can change one of the cw setting in the example config to be true/false